service/controller/app/v1/resource/chartoperator/create.go

```diff
@@ -0,0 +1,46 @@
+package chartoperator
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/app-operator/service/controller/app/v1/controllercontext"
+	"github.com/giantswarm/app-operator/service/controller/app/v1/key"
+)
+
+func (r Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	cr, err := key.ToCustomResource(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+	cc, err := controllercontext.FromContext(ctx)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	// Check whether tenant cluster has a chart-operator helm release yet.
```

Comment:
Yes, we only want to bootstrap chart-operator in TCs. But we also need to process app CRs in the CP for VOO. Is this handled?

If not I think we should cancel the resource if InCluster is true. In the CP chart-operator is deployed by draughtsman.

Reply:
I didn't know that it deployed by draughtsman, so I can skip this resource on inCluster mode as you said. Thanks for pointing it out.

Reply:
@rossf7 I found out this results in an error from e2etest, since we are not installing fresh chart-operator from `simple-test`. So I revered this change. Nevertheless, we will always look into helm charts list in CP, then we will skip installing chart-operator whenever they are already there. WDYT?

Reply:
OK cool. Good point that there is the chart-operator release created by draughtsman.